src/components/atoms/Toggler/Toggler.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "classnames";
+import React, {
+  DetailedHTMLProps,
+  forwardRef,
+  InputHTMLAttributes,
+} from "react";
+import { v4 as uuidv4 } from "uuid";
+
+import "./Toggler.scss";
+
+export interface TogglerProps
+  extends DetailedHTMLProps<
+    InputHTMLAttributes<HTMLInputElement>,
+    HTMLInputElement
+  > {
+  title?: string;
+  containerClassName?: string;
+  inputClassName?: string;
+}
+
+const Toggler: React.ForwardRefRenderFunction<HTMLDivElement, TogglerProps> = (
```

Comment:
I did the component based on recently implemented `InputField`. Should I remove the ref?

![image](https://user-images.githubusercontent.com/63194656/116603580-bd58b400-a935-11eb-8094-5a61787e47f0.png)

Reply:
Searching my current branch, I can only see 2 usages of `InputField`, and neither of them make use of the `ref`; so i'm not sure what the original intent of it is/was without going back and looking at the PR where it was introduced.

Reply:
History: 

- https://github.com/sparkletown/sparkle/commits/staging/src/components/atoms/InputField/InputField.tsx

Was added in https://github.com/sparkletown/sparkle/pull/1306 to resolve https://github.com/sparkletown/internal-sparkle-issues/issues/615

Specifically in this commit: https://github.com/sparkletown/sparkle/pull/1306/commits/c137aec001cc7119da4274f039102d1d65e2c96b

I talk about inlining the `forwardRef` in https://github.com/sparkletown/sparkle/pull/1306/files#r611303189 + https://github.com/sparkletown/sparkle/pull/1306/files#r611300489

A quick search of those issues doesn't seem to show up any other comments/etc for why it was added there.

Off the top of my head, my guess/gut feel is that it might have been intended to be passed directly to the `input` element itself, for getting a reference to that; as that would generally make more sense to me than attaching it to the `div`. Not 100% on that though.

---

@sunny-viktoryia Do you remember why you added `forwardRef` + attached the `ref` to the `div` in `InputField` by chance?

Reply:
Looks like I might have misunderstood you. Yeah, I added `ref` based on your comment. 
https://github.com/sparkletown/sparkle/pull/1306#discussion_r607727239

I thought we can access `<input />` based on parent div ref if we want.

Reply:
created a pull request that moves ref from `div` to `input`
https://github.com/sparkletown/sparkle/pull/1353

sorry for the inconveniences caused.

Reply:
> @sunny-viktoryia No worries. Thanks for making the fix + pointing out where it was originally discussed (I thought my memory was going crazy :p)
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/sparkle/pull/1343#discussion_r623639120_

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1299/commits/13d2c8b1bc477d69ee9814144c4fd09d3e1383a6